hoomd/md/PotentialPairDPDThermo.h

```diff
@@ -184,7 +184,7 @@ template<class evaluator> void PotentialPairDPDThermo<evaluator>::computeForces(
             assert(typej < this->m_pdata->getNTypes());
 
             // apply periodic boundary conditions
-            dx = box.minImage(dx);
+            box.minImage(dx, dv);
```

Comment:
@mphoward `minImage` is the method I was referring to that returns a value in two out of four overloads, but not in the other two. I understand that C++ lacks a natural syntax for multiple return value unpacking, so I don't have a better suggestion.